app/components/Header.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+import { Link as ChakraLink } from "@chakra-ui/next-js";
+import { Box, Grid, Stack, IconButton } from "@chakra-ui/react";
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import { useState } from "react";
+import styled from "@emotion/styled";
+
+export default function Header() {
+  const [hamburger, setHamburger] = useState(false);
+
+  const pathname = usePathname();
+
+  const handleHamburger = () => {
+    setHamburger(!hamburger);
+  };
+
+  return (
+    <Stack>
+      <Grid templateColumns={"500px 1fr 100px"}>
+        <Box p="12px">
+          <Image
+            src="/images/kenworthy-logo.jpg"
+            alt="Kenworthy Machine"
+            width={500}
+            height={500}
+          />
+        </Box>
+        <Box
+          display="flex"
+          alignItems="center"
+          justifyContent="flex-end"
+          gap="24px"
+        >
+          <ChakraLink
+            href="/"
+            borderBottom={
+              pathname === "/" ? "solid var(--primary-red) 2px" : "none"
+            }
+            variant="navigation"
+          >
+            Home
+          </ChakraLink>
+
+          <ChakraLink
+            href="/request-quote"
+            borderBottom={
+              pathname === "/request-quote"
+                ? "solid var(--primary-red) 2px"
+                : "none"
+            }
+            variant="navigation"
+          >
+            Request a Quote
+          </ChakraLink>
+
+          <IconButton aria-label="Hamburger menu" onClick={handleHamburger}>
+            <Image
+              src="/images/hamburger-menu.png"
+              alt="Hamburger Menu"
+              width={40}
+              height={40}
+            />
+          </IconButton>
```

Comment:
![Screenshot 2024-01-22 at 2 49 29 PM](https://github.com/wavecrestweb/kenworthy-machine/assets/115492619/e0183a82-449e-41fa-ada4-36288f71240c)
There's an outline around the hamburger icon in my browser - maybe from the custom themes set up for for buttons? I can't quite figure out what's causing it. I would suggest adding a hover and focus property for accessibility and removing the outline.